README.md

```diff
@@ -15,11 +17,12 @@ The direct user of OpenFunction Context is the OpenFunction Builder (Go). The fo
 
 > You can get the corresponding version compatibility information from the specific builder repository. Here we use the OpenFunction Go (v1.15) builder as an example.
 
-| OpenFunction | Context | Builder (Go)                                |
-| ------------ | ------- | ------------------------------------------- |
+| OpenFunction | Context | Builder (Go)                                 |
+| ------------ | ------- | -------------------------------------------- |
 | v0.3.*       | v0.1.0  | v0.2.2 (openfunction/builder-go:v0.2.2-1.15) |
 | v0.4.*       | v0.2.0  | v0.3.0 (openfunction/builder-go:v0.3.0-1.15) |
 | v0.5.*       | v0.2.0  | v0.4.0 (openfunction/builder-go:v0.4.0-1.15) |
+| v0.6.*       | v0.3.0  | v0.5.0+                                      |
```

Comment:
The build could be [v2-1.16](https://hub.docker.com/layers/187319070/openfunction/builder-go/v2-1.16/images/sha256-c2b1fbaea4e42ad988c2901125c853cb826e94c233b325d710ce2577b7441ee9?context=repo) , This image could be updated after recompiling the builder

Reply:
ok